core/src/main/java/org/commonjava/maven/ext/core/impl/RelocationManipulator.java

```diff
@@ -128,13 +161,66 @@ private boolean apply( final Project project, final Model model ) throws Manipul
             result |= updateDependencies( relocations, project.getAllResolvedProfileDependencies( session ).get( profile ) );
 
         }
+
+        result |= updateAllPlugins( model, project, relocations );
+
+        return result;
+    }
+
+    private boolean updateAllPlugins( final Model model, final Project project, final WildcardMap<ProjectVersionRef> relocations ) throws ManipulationException
+    {
+        Map<String, Plugin> pluginMap = getManagedPluginMap( model );
```

Comment:
Why can't we use https://github.com/release-engineering/pom-manipulation-ext/blob/master/common/src/main/java/org/commonjava/maven/ext/common/model/Project.java#L381 ?

Reply:
I'll change it to use that and remove the older code from the file. I believe that code is not handling profiles anyway.

Reply:
Good point ; there are specialised functions in the Project class to return fully resolved set from profiles as well.

Reply:
OK, it is agreed to make this change.